internal/templates/home.go

```diff
@@ -0,0 +1,145 @@
+package templates
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/writhe/kudosaurus"
+)
+
+// Kudo - kudo view block
+func Kudo(kudo kudosaurus.Kudo) string {
+	template := `
```

Comment:
This templates are tied directly to slack format - this all should be in slack package.